app_dart/lib/src/model/firestore/task.dart

```diff
@@ -284,6 +322,7 @@ class Task extends Document {
   String toString() {
     final StringBuffer buf = StringBuffer()
       ..write('$runtimeType(')
+      ..write('$kTaskBuildNumberField: $buildNumber')
```

Comment:
```suggestion
      ..write(', $kTaskBuildNumberField: $buildNumber')
```

Reply:
This would output Task(, 123, etc..). Is that the intent?